src/wizard/dotnet/google-cloud-functions.md

```diff
@@ -0,0 +1,107 @@
+---
+name: Google Cloud Functions
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/google-cloud-functions/
+support_level: production
+type: framework
+---
+
+Install the **NuGet** package:
+
+Package Manager:
+
+```shell
+Install-Package Sentry.Google.Cloud.Functions -Version {{ packages.version('sentry.dotnet.google-cloud-function') }}
+```
+
+Or .NET Core CLI:
+
+```shell
+dotnet add package Sentry.Google.Cloud.Functions -v {{ packages.version('sentry.dotnet.google-cloud-function') }}
+```
+
+Add Sentry to `Function` class through the `FunctionsStartup`:
+
+
+```csharp
+// Add the following line:
+[assembly: FunctionsStartup(typeof(SentryStartup))]
+
+public class Function : IHttpFunction
+{
+    public Task HandleAsync(HttpContext context)
+    {
+        // Your function code here. 
+    }
+}
+```
+
+Additionally, you'll need to set up your `Sentry` settings on `appsettings.json`:
+
+```json
+{
+  "Sentry": {
+    "Dsn": "___PUBLIC_DSN___",
+    // Sends Cookies, User Id when one is logged on and user IP address to sentry. It's turned off by default.
+    "SendDefaultPii": true,
+    // Enable tracing.
+    "TracesSampleRate": 1.0,
```

Comment:
This doesn't get any transactions in automatically since we're not adding the middleware yet